docs/DeveloperGuide.md

```diff
@@ -365,20 +365,24 @@ Refer to the guide [here](./DevOps.md)
 
 **Target user profile**:
 
+* university students
 * take multiple modules
-* want to easily track their weekly calendar
-* want to be notified about important academic dates coming up
-* want to have their personal todos & events integrated with their academic calendar
+* want to easily know the events lined up in their week ahead
+* prefer to have everything in one application
+* want to have their personal tasks integrated with their academic calendar
 * want to access related contacts when they browse a module
-* want to keep track of modules' details (grading scheme, module's knowledge coverage)
+* want to keep track of module details (relevant lessons, tasks, and contacts)
+* want to manage their modules, lessons, tasks, and contacts without an internet connection
```

Comment:
Hmm I think no need for this, because I think our app is good on its own and even with Internet there is nothing more we can improve 😄